mathesar_ui/src/systems/record-selector/RecordSelectorContent.svelte

```diff
@@ -149,7 +149,7 @@
   {/if}
 </div>
 
-<style>
+<style lang="scss">
```

Comment:
A few comments here:

- This `lang="scss"` is unnecessary in this block.

- I'm fine with using SCSS in cases where we have a lot of nesting. But otherwise, I prefer sticking to the vanilla CSS because the svelte tooling supports it better. For example, if you have unused CSS, you'll get a pretty clear warning in your editor showing you exactly where it is with vanilla CSS. With SCSS, you get a tiny squiggle over the closing `<` character of the opening `<style>` tag. If we don't _need_ SCSS, then I'd prefer to stick to vanilla CSS.

- **But** for this case, I cleaned this code up while I was resolving git conflicts when basing my #2185 on this code. So please don't make any more changes within `src/systems/record-selector` unless absolutely necessary.